config.py

```diff
@@ -50,6 +50,77 @@ def __init__(self, use_baseline):
             self.max_ep_len = self.batch_size
 
 
+class StandardLVNetwork(object):
+    def __init__(self, env_name, use_baseline):
+        self.env_name = env_name
+
+        # output config
+        baseline_str = 'baseline' if use_baseline else 'no_baseline'
+        self.output_path = "results/{}-{}/".format(self.env_name, baseline_str)
+        self.model_output = self.output_path + "model.weights/"
+        self.log_path = self.output_path + "log.txt"
+        self.plot_output = self.output_path + "scores.png"
+        self.record_path = self.output_path
+        self.record_freq = 5
+        self.summary_freq = 1
+
+        # model and training config
+        self.num_batches = 100  # number of batches trained on
+        self.batch_size = 1000  # number of steps used to compute each policy update
+        self.max_ep_len = 60  # maximum episode length
+        self.learning_rate = 3e-2
+        self.gamma = 0.9  # the discount factor
+        self.use_baseline = use_baseline
+        self.normalize_advantage = True
+
+        self.remove_q = True
+        self.clear_loads_sgen = False
+        self.clear_gen = True
+
+        # environment generation
+        self.static_feeds_new = None  # Acts how static_feeds does in the 6BusPOC config
+        # self.static_feeds_new = {
+        #     3: -10 * np.ones(self.max_ep_len),
+        #     6: -10 * np.ones(self.max_ep_len),
+        #     4: 10 * np.ones(self.max_ep_len),
+        #     7: 10 * np.ones(self.max_ep_len)
+        # }
+
+        # Fill static_feeds with the loads and static generators that ship with the network
+        if self.static_feeds_new is None:
+            self.static_feeds = {}
+        else:
+            self.static_feeds = self.static_feeds_new.copy()
```

Comment:
I'm not following lines 81-93. It seems like a roundabout way to set `self.static_feeds = {}`.